src/test/java/com/github/themeetgroup/kafka/connect/rabbitmq/sink/RabbitMQSinkTaskTest.java

```diff
@@ -13,33 +14,42 @@
 import java.util.Map;
 import java.util.List;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import java.util.concurrent.TimeUnit;
 
 
+@Testcontainers
 class RabbitMQSinkTaskTest {
 
     @Test
-    public void testStart() throws IOException, TimeoutException {
+    void testStart() throws IOException, TimeoutException, InterruptedException {
+        TimeUnit.SECONDS.sleep(20);
 
         String queue = "test_queue";
         String routingKey = "test_routing_key";
-        String exchange = "exchange";
 
         RabbitMQSinkTask sinkTask = new RabbitMQSinkTask();
-        createExchangeAndQueue(exchange, queue);
-        sinkTask.start(createSettings(exchange, queue, routingKey));
+        createQueue(queue);
+        sinkTask.start(createSettings(queue, routingKey));
         assertTrue(getBindings(queue).stream()
-                .filter(n -> n.get("source").equals(exchange))
+                .filter(n -> n.get("source").equals("data"))
                 .map(v -> v.get("routing_key"))
                 .anyMatch(t -> t.equals(routingKey)));
     }
 
+    @Container
+    private static DockerComposeContainer environment =
+            new DockerComposeContainer(new File("src/test/java/com/github/themeetgroup/kafka/connect/rabbitmq/sink/docker-compose.yaml"));
```

Comment:
docker-compose file can be moved to resources, and then loaded with `new File(getClass().getClassLoader().getResource("docker-compose.yaml"))`